client/query.go

```diff
@@ -131,8 +131,9 @@ func sdkErrorToGRPCError(resp abci.ResponseQuery) error {
 // or an error if the query fails.
 func (ctx Context) query(path string, key tmbytes.HexBytes) ([]byte, int64, error) {
 	resp, err := ctx.queryABCI(abci.RequestQuery{
-		Path: path,
-		Data: key,
+		Path:   path,
+		Data:   key,
+		Height: ctx.Height,
```

Comment:
This is to keep convention with grpc query

https://github.com/cosmos/cosmos-sdk/blob/b67c4b3e861beb2e7d048ebf93f111278e6f2d5c/client/grpc_query.go#L75-L79